src/Engine/System/RenderSystem.cpp

```diff
@@ -78,6 +85,21 @@ void RenderSystem::update(float dt) {
         ImGui::Render();
     }
 #endif
+
+    // Make it so that rendering is done to the computer screen
+    glBindFramebuffer(GL_FRAMEBUFFER, 0);
+
+    // Reset rendering display
+    glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+    glClearColor(0.2f, 0.3f, 0.4f, 1.f);
+
+    glViewport(0, 0, size.x, size.y);
+
+    squareShader->bind();
+    // The second parameter is passed by reference (not by pointer),
+    // hence the funny pointer business
+    squareShader->render(nullptr, *((Vector<Component *> *) nullptr));
```

Comment:
Most likely this vector of components will not need to be passed to every shader in the future. For now, potentially a better way of doing this would be to define an empty static vector and pass a reference to that.

Reply:
How come passing a dummy vector would be better?

Reply:
I agree with Brenan. This shouldn't make a difference, and it'll be patched with future RenderSystem updates anyways.

Reply:
To clarify, I was not arguing in favor for the current implementation -- I was asking why the vector would be better

Reply:
Neither solution is good, but dereferencing a nullptr is pretty sketch, and I've been known to do a sketchy thing from time to time. An empty vector would still be a valid object, just useless. This is invalid and will cause memory errors if touched